src/integration_fleets_tutorial.md

```diff
@@ -0,0 +1,386 @@
+
+> The code giving below serves as an example for RobotClientAPI.py implementation. Code can be found [here](https://github.com/thedevmanek/rmf_demos/tree/main/rmf_demos_fleet_adapter/rmf_demos_fleet_adapter).
+
+
+## Understanding `config.yaml` file
+```yaml
+# FLEET CONFIG =================================================================
+# RMF Fleet parameters
+
+rmf_fleet:
+  name: "DeliveryBot"
+  fleet_manager:
+    prefix: "http://127.0.0.1:8000"
+    user: "some_user"
+    password: "some_password"
+  limits:
+    linear: [0.4, 0.2] # velocity, acceleration
+    angular: [0.3, 0.35] # velocity, acceleration
+  profile: # Robot profile is modelled as a circle
+    footprint: 0.5 # radius in m
+    vicinity: 0.6 # radius in m
+  reversible: False # whether robots in this fleet can reverse
+  battery_system:
+    voltage: 24.0 # V
+    capacity: 40.0 # Ahr
+    charging_current: 26.4 # A
+  mechanical_system:
+    mass: 80.0 # kg
+    moment_of_inertia: 20.0 #kgm^2
+    friction_coefficient: 0.20
+  ambient_system:
+    power: 20.0 # W
+  tool_system:
+    power: 760.0 # W
+  recharge_threshold: 0.20 
+  recharge_soc: 1.0 
+  publish_fleet_state: True
+  account_for_battery_drain: True
+  task_capabilities: 
+    loop: True
+    delivery: False
+    clean: False
+    finishing_request: "nothing" # [park, charge, nothing]
+
+# DeliveryBot CONFIG =================================================================
+
+robots:
+  deliverybot1:
+    robot_config:
+      max_delay: 10.0 
+    rmf_config:
+      robot_state_update_frequency: 0.5
+      start:
+        map_name: "L1"
+        # waypoint: "charger_deliverybot1" # Optional
+        # orientation: 0.0 # Optional, radians
+        waypoint: null
+        orientation: null
+      charger:
+        waypoint: "charger_deliverybot1"
+reference_coordinates:
+  rmf: [[20.33, -3.156],
+        [8.908, -2.57],
+        [13.02, -3.601],
+        [21.93, -4.124]]
+  robot: [[59, 399],
+        [57, 172],
+        [68, 251],
+        [75, 429]]
+```
+
+  
+
+- `rmf_fleet` information about the fleet and the bots in the fleet.
+
+- `fleet_manager` prefix, username and password
+
+- `limits` maximum values for linear and angular accelerations and velocities.
+
+- `profile` footprint and vicinity.
+
+- `reversible` a flag that can enable/disable reverse traversal in the robot.
+
+- `battery_system` information about the battery
+
+- `recharge_threshold` sets a value for minimum charge below which the robot cannot operate.
+
+- `recharge_soc` the maximum level to which robots should be charged.
+
+- `task_capabilities` the capabilities of the robot
+
+- `finishing_request` can be set to `park`, `charge` or `nothing`
+
+- `robots` information about all the types of robots in this case there is only one robot i.e. delivery bot
```

Comment:
@thedevmanek Could you change the `config.yaml` file used to the one linked in the above comment? So that users can understand how multiple robots are configured. This way you won't need to modify the robot name below.

Reply:
Done